### PR TITLE
Generate new bucket name if it already exists

### PR DESCRIPTION
**Pull request by legionus**

### CODE REVIEW

File: pkg/storage/gcs/gcs.go [hunk 1]
```diff
@@ -54,9 +54,22 @@ func (d *driver) CompleteConfiguration() error {
 			return err
 		}
 
-		err = client.Bucket(d.Config.Bucket).Create(ctx, projectID, nil)
-		if err != nil {
-			return err
+		d.Config.Bucket = "image-registry"
```
[the remark this answers is not in the document]
Comment (legionus) [reply pkg/storage/gcs/gcs.go]: @bparees the first attempt to create a pretty name (a new installation?).

[the remark this answers is not in the document]
Comment (bparees) [reply pkg/storage/gcs/gcs.go]: let's just be consistent and always add the suffix.  if you're sharing your s3 account between cluster installs this is just going to fail most of the time.

[the remark this answers is not in the document]
Comment (legionus) [reply pkg/storage/gcs/gcs.go]: @bparees by the way a small offtopic: With the movement to operators, it seems that we made it impossible to use gcs to start containers and at the same time store the images in s3. With ansible this was possible.

[the remark this answers is not in the document]
Comment (legionus) [reply pkg/storage/gcs/gcs.go]: > let's just be consistent and always add the suffix.

@bparees fixed.

[the remark this answers is not in the document]
Comment (bparees) [reply pkg/storage/gcs/gcs.go]: > With the movement to operators, it seems that we made it impossible to use gcs to start containers and at the same time store the images in s3. With ansible this was possible.

one of many goals is to reduce the number of configurations we support.  I'm fine with this.  If it's needed we can allow it in the future.

**Pull request merged**